ScienceDirect.js

```diff
@@ -78,6 +78,15 @@ function getPDFLink(doc, onDone) {
 		return;
 	}
 	
+	// when using a partnered institution network access, construct the the PDF URL directly
```

Comment:
→ `// When using partnered institution network access, construct the PDF URL directly`